shared/js/background/trackers.es6.js

```diff
@@ -220,10 +229,17 @@ function checkTrackersWithParentCompany (blockSettings, url, currLocation) {
         // to pull off subdomains until we either find a match or have no url to check.
         // Ex: x.y.z.analytics.com would be checked 4 times pulling off a subdomain each time.
         url.shift()
-        return checkTrackersWithParentCompany(blockSettings, url, currLocation)
+        return checkTrackersWithParentCompany(blockSettings, url, siteDomain, request)
     }
 }
 
+function requestMatchesRule (request, rule) {
+    if (rule.exec(request.url)) {
```

Comment:
could this function be reduced to the following?

`return rule.exec(request.url)`

Reply:
👍